src/pybind/mgr/dashboard/frontend/package.json

```
@@ -10,18 +10,19 @@
     "start": "npm run env_build && ng serve --host 0.0.0.0",
     "build": "export _locale=${LOCALE:-$npm_package_config_locale}; ng build --outputPath=dist/${_locale} --i18nFile=src/locale/messages.${_locale}.xlf --i18nLocale=${_locale}",
     "prebuild": "npm run env_build",
-    "build:en-US": "LOCALE=en-US npm run build",
-    "build:de-DE": "LOCALE=de-DE npm run build",
-    "build:es-ES": "LOCALE=es-ES npm run build",
-    "build:pt-BR": "LOCALE=pt-BR npm run build",
-    "build:it-IT": "LOCALE=it-IT npm run build",
-    "build:cs": "LOCALE=cs npm run build",
-    "build:fr-FR": "LOCALE=fr-FR npm run build",
-    "build:id-ID": "LOCALE=id-ID npm run build",
-    "build:ja-JP": "LOCALE=ja-JP npm run build",
-    "build:pl-PL": "LOCALE=pl-PL npm run build",
-    "build:zh-CN": "LOCALE=zh-CN npm run build",
-    "build:zh-TW": "LOCALE=zh-TW npm run build",
+    "build:cs": "LOCALE=cs npm run build-i18n",
```

`build-i18n` has been removed, please use `build`
```suggestion
    "build:cs": "LOCALE=cs npm run build",
```